lib/popup.ts

```diff
@@ -182,6 +182,13 @@ export class PopupControl<T extends IPopupOptions = IPopupOptions> extends Dispo
             case 'click':
               dom.onElem(triggerElem, 'click', () => this.toggle());
               break;
+            case 'contextmenu':
+              dom.onElem(triggerElem, 'contextmenu', (ev) => {
+                ev.preventDefault(); // Prevent the default browser contextmenu behavior
+                this.open();
+              });
+              dom.onElem(triggerElem, 'click', () => this.close());
```

Comment:
Lets remove it if it's unneeded (since click anywhere closes it). Maybe just add a comment that an explicit handle is unneeded.

Reply:
No, it's not unneeded. In my tests, clicking on the element did not register as "click anywhere", so I had to add an explicit handler. I could dig into the definition of "anywhere", but this seemed like an easy solution.

Reply:
Ah, makes sense. Could you add a comment then that popups are normally close by clicks anywhere except triggerElem (and popup content), but in this case we want clicks on triggerElem to close it too.